support-frontend/assets/helpers/storage/abandonedBasketCookies.ts

```diff
@@ -38,11 +39,13 @@ export function useAbandonedBasketCookie(
 		region,
 	};
 	useEffect(() => {
-		cookie.set(
-			ABANDONED_BASKET_COOKIE_NAME,
-			JSON.stringify(abandonedBasket),
-			COOKIE_EXPIRY_DAYS,
-		);
+		if (inAbandonedBasketVariant) {
```

Comment:
has to be at this level because to follow rules of hooks and not call useEffect or useAbandonedBasket conditionally